.travis.yml

```diff
@@ -1,34 +1,53 @@
 language: python
+sudo: false
+dist: bionic
+
 cache: pip
 
+python:
+  - "3.6"
+  - "3.7"
+
 addons:
   apt:
     packages:
       - sqlite3
+      - fping
       - gdal-bin
+      - openssl
       - libproj-dev
       - libgeos-dev
       - libspatialite-dev
+      - spatialite-bin
+      - libsqlite3-mod-spatialite
 
-python:
-  - "3.6"
+services:
+  - docker
+  - redis-server
 
 branches:
   only:
     - master
 
 before_install:
-  - pip install -U pip wheel setuptools
+  - docker run -d --name influxdb -p 8086:8086 influxdb
+  - docker exec -it influxdb influx -execute 'CREATE DATABASE openwisp2'
+  - pip install -U pip wheel setuptools pyasn1
   - pip install --no-cache-dir -U -r requirements-test.txt
```

Comment:
let's remove `--no-cache-dir` for consistency with other modules. If you find this in other modules let me know so we can remove it. I added it at some point in the past in which I was going crazy with the travis cache. Then they added a way to clear the cache from the web app.

Reply:
I took reference from `openwisp-firmware-upgrader` and it is still present [there](https://github.com/openwisp/openwisp-firmware-upgrader/blob/6f798d429dffbf0d4a3503b4b3d1cb293706b2e7/.travis.yml#L34). Should I create a PR for that?